tests/testdata/projects/package-managers/go/missing-context/go.mod

```diff
@@ -0,0 +1,160 @@
+module missing_context
+
+go 1.22
+
+require (
+	cloud.google.com/go v0.38.0 // indirect
+	github.com/Azure/azure-sdk-for-go v40.3.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.10.0 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.8.2 // indirect
+	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2 // indirect
+	github.com/Azure/go-autorest/autorest/azure/cli v0.3.1 // indirect
```

Comment:
Please only include the needed dependency for the test to reduce size and test times